src/main/java/duke/ui/MainWindow.java

```diff
@@ -0,0 +1,74 @@
+package duke.ui;
+
+import duke.Duke;
```

Comment:
Perhaps the import statements can be grouped by their packages? I noticed this in other areas as well.